### ISSUE TITLE
`dsc` does not work for starred commands e.g. `\section*{First Section}`

### ISSUE THREAD

**Issue by kiryph** [lervag/vimtex#691]
Can anybody confirm that `dsc` does not delete the surrounding command for following example:
```tex
\section*{First section}
```
In this case the cursor should be allowed to be anywhere on the line and `dsc` should delete the surrounding command to change the line to
```tex
First Section
```
However, if the cursor is on `\section` it just deletes `\section`. Otherwise it won't do anything.
Also affected is `csc`.

I can confirm that it works for
```tex
\section{First section}
```

**Comment by kiryph** [comment]
Fixed with PR #693.

**Comment by lervag** [comment]
Thanks!
